lib/rspec/core/option_parser.rb

```diff
@@ -140,8 +140,18 @@ def parser(options)
           options[:full_backtrace] = true
         end
 
-        parser.on('-c', '--[no-]color', '--[no-]colour', 'Enable color in the output.') do |o|
-          options[:color] = o
+        parser.on('-c', '--color', '--colour', 'Deprecated: enable color in the output.') do |_o|
+          options[:color] = true
+        end
```

Comment:
The more I think about it, the more I lean towards removing it.  We need to not break when people pass `-c` or `--color` for backwards compatibility but we don't really want to waste new user's time having them learn about it when they run `rspec --help`.

In the absence of anyone arguing to keep it in the help, I'm going to say let's remove it.